docker-compose/docker-compose.yaml

```diff
@@ -180,6 +183,18 @@ services:
     volumes:
       - ./vault/init.sh:/vault/scripts/init.sh
 
+  opa-server:
+    image: openpolicyagent/opa:latest
+    container_name: opa-server
+    profiles:
+      - identity
```

Comment:
I would create a separate profile name for it, e.g. opa and replace identity with it
so that, as a user, I don't put additional burden on my machine if I don't use dynamic policies